std/typecons.d

```diff
@@ -4175,8 +4216,7 @@ private static:
             {
                 preamble ~= "alias self = " ~ name ~ ";\n";
                 if (WITH_BASE_CLASS && !__traits(isAbstractFunction, func))
-                    //preamble ~= "alias super." ~ name ~ " parent;\n"; // [BUG 2540]
-                    preamble ~= "auto parent = &super." ~ name ~ ";\n";
+                    preamble ~= "alias parent = AliasSeq!(__traits(getMember, super, \"" ~ name ~ "\"))[0];";
```

Comment:
A solution would be to use Identity alias from std.meta. The problem is that it is private for some reason. I'd say __traits would look nicer being encapsulated in Identity alias rather than AliasSeq template.